itest/lnd_channel_graph_test.go

```diff
@@ -335,17 +335,30 @@ func testGraphTopologyNtfns(ht *lntest.HarnessTest, pinned bool) {
 
 	// For the final portion of the test, we'll ensure that once a new node
 	// appears in the network, the proper notification is dispatched. Note
-	// that a node that does not have any channels open is ignored, so first
-	// we disconnect Alice and Bob, open a channel between Bob and Carol,
-	// and finally connect Alice to Bob again.
-	ht.DisconnectNodes(alice, bob)
+	// that a node that does not have any channels open is ignored, so
+	// first we disconnect Alice and Bob, open a channel between Bob and
+	// Carol, and finally connect Alice to Bob again.
+	ht.DisconnectNodes(bob, alice)
```

Comment:
Ah, I think we relied on the reconnect retry backoff before, which is obviously wonky. So nice to address this properly :tada: